conversion_proxy/src/lib.rs

```diff
@@ -107,15 +122,21 @@ impl ConversionProxy {
             env::prepaid_gas(),
             MIN_GAS
         );
+        assert_eq!(
+            currency, "USD",
+            "Only payments denominated in USD are implemented for now"
```

Comment:
Why bother asserting this?

Reply:
@MantisClone Nothing else prevents from trying to pay with another denomination, not even a missing dictionary key.

Reply:
Wouldn't paying with another denomination be fine? And if the dictionary key was missing wouldn't it cause an error and revert?

Reply:
We only have [NEAR/USD](https://app.switchboard.xyz/near/mainnet) on mainnet and testnet, so another denomination would not be fine. There is no dictionary since we only support USD for now, hence the assertion.

Reply:
Would you remove the assertion if a new NEAR price feed were introduced like NEAR/EUR? Or if we adjusted the contract to support price feed paths like NEAR/USD -> USD/ETH?

Reply:
We should remove the assertion when a NEAR/[fiat] price feed is available and once we have implemented some kind of dictionary (and maybe some administration?)

Reply:
Okay :+1: